api/main.go

```diff
@@ -63,12 +78,17 @@ func Register(c *gin.Context) {
 }
 
 func main() {
+	err := godotenv.Load()
+	if err != nil {
+		log.Fatal("Error loading .env file")
+	}
+
 	CreateTables(context.Background())
 
 	// Create gin server
 	r := gin.Default()
 	r.Use(cors.New(cors.Config{
-		AllowOrigins:     []string{"http://localhost:3000", "http://localhost:4000"},
+		AllowOrigins:     []string{"*"},
```

Comment:
I had to add this back to the code below to get it to work in Windows
```suggestion
		AllowOrigins:     []string{"http://localhost:3000", "http://localhost:4000"},
```

Reply:
Just tried it out again locally, for some reason it works fine through Firefox, but not Chrome :shrug: